Sources/HTTPParser/http_parser.swift

```diff
@@ -845,15 +861,15 @@ private func IS_HEADER_CHAR(_ ch: UInt8) -> Bool {
   return (ch == CR || ch == LF || ch == 9 || (ch > 31 && ch != 127))
 }
 
-private func STRICT_TOKEN(_ c: UInt8) -> UInt8 {
+private mutating func STRICT_TOKEN(_ c: UInt8) -> UInt8 {
```

Comment:
Mutating is required on 10 or so functions if using `struct http_parser`.  This is not required using `class http_parser`.

The compiler error is:
`Mark method 'mutating' to make 'self' mutable`